Apps/W1/ShopifyConnector/app/Products/Tables/ShpfyInventoryItem.Table.al

```diff
@@ -0,0 +1,81 @@
+/// <summary>
+/// Table Shpfy Inventory Item (ID 30126).
+/// </summary>
+table 30126 "Shpfy Inventory Item"
+{
+    Access = Internal;
+    Caption = 'Shopify Inventory Item';
+    DataClassification = CustomerContent;
+
+    fields
+    {
+        field(1; Id; BigInteger)
+        {
+            Caption = 'Id';
+            DataClassification = CustomerContent;
+        }
+        field(2; "Variant Id"; BigInteger)
+        {
+            Caption = 'Variant Id';
+            DataClassification = CustomerContent;
+        }
+        field(3; "Country of Origin"; Text[50])
```

Comment:
please rename field to: "Country/Region of Origin"

Reply:
Updated with commit "Object Renaming"